site/public/css/diff-viewer.css

```diff
@@ -8,24 +8,37 @@
 
 .diff-code {
     font-family: monospace;
-    color: var(--btn-text-black);
+    color: var(--diff-char);
+    background-color: var(--default-white);
     line-height: 1.8;
 }
 
 .diff-code .row {
     width: 100%;
 }
 
-.diff-code .bad {
-    background-color: var(--alert-background-bronze);
+.diff-code .bad-actual {
+    background-color: var(--diff-background-actual);
+    color: var(--diff-char);
+}
+
+.diff-code .bad-expected {
+    background-color: var(--diff-background-expected);
+    color: var(--diff-char);
 }
 
 .highlight-hover {
     background-color: var(--standard-light-blue-gray) !important;
 }
 
-.highlight-char {
-    background-color: var(--focus-danger-red);
+.highlight-char-actual {
+    background-color: var(--diff-char-background-actual);
+    font-weight: 900;
```

Comment:
I don't think we agreed on adding the boldness for highlighted char.

Reply:
The demo I gave the other day included making highlighted characters bold as well. 
May be its my screenshot fault that you missed it. 
We can discuss in today's meet :)